### PR TITLE
pd-ctl: fix hot region show

### LINKED ISSUE #6649: pd-ctl: hot region show unexpect info
### What did you do?

`pd-ctl hot read`

### What did you expect to see?

all info

### What did you see instead?

Image

### What version of PD are you using (`pd-server -V`)?

v6.5.3

### PR DESCRIPTION
**Pull request by lhy1024**
### What problem does this PR solve?

Issue Number: Close #6649

### What is changed and how does it work?

```commit-message
```

### Check List

Tests

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)
Image
Image

### Release note

```release-note
None.
```

### CODE REVIEW

**Review by CabinfeverB — APPROVED** [review]

**Comment by nolouch** [comment]
Does it affect `packHistoryHotRegions` for tidb history hot region table?

**Comment by lhy1024** [comment]
> `packHistoryHotRegions`

no, it gets data from `Handler.packHotRegions`

**Review by nolouch — APPROVED** [review]

File: pkg/schedule/coordinator.go [hunk 1]
```diff
@@ -519,11 +519,28 @@ func (c *Coordinator) GetHotRegionsByType(typ statistics.RWType) *statistics.Sto
...
+				h := &s[i][j].Stats[k]
+				region := c.cluster.GetRegion(h.RegionID)
+				if region != nil {
+					h.IsLearner = core.IsLearner(region.GetPeer(h.StoreID))
+				}
+				switch typ {
+				case statistics.Write:
+					if region != nil {
+						h.LastUpdateTime = time.Unix(int64(region.GetInterval().GetEndTimestamp()), 0)
+					}
```
Comment (HuSharp) [inline pkg/schedule/coordinator.go:535]: Why not use store heartbeat to unify?

Comment (lhy1024) [reply pkg/schedule/coordinator.go:535]: We collect write hot info by region heartbeat

**Pull request merged**